src/utils/social-media.ts

```diff
@@ -21,6 +23,7 @@ export const socialMediaHandles = {
 };
 
 export const socialMediaDescriptions = {
+  [SocialMedia.discord]: 'Join our community on discord to chat.',
```

Comment:
Would `Join our discord community to make new friends and interact with one another.` be good?

Reply:
done

Reply:
@mrbusysky it seems like the logo is still white